github/index.js

```diff
@@ -91,7 +91,7 @@ class GitHub {
         throw e
       }
     } catch (e) {
-      console.warn('Unable to retrieve username or membership status:', e)
+      console.warn('Unable to retrieve username or membership status:', e.message)
```

Comment:
changed this because API logs were filling up with a huge `got` error object (useless) everytime we checked if a user was an admin (in staging, where IDs are screwy).